src/muse/carbon_budget.py

```diff
@@ -336,14 +336,30 @@ def bisection(
     price_too_high_threshold: float = 10,
     fitter: Text = "slinear",
 ) -> float:
-    # to be set as moving value in superloop before emissions start increasing
+    """Applies bisection algorithm to escalate carbon price and meet the budget.
```

Comment:
I feel this needs a more detailed description of what this algorithm is about. Probably, also the other algorithm needs it, but given that we are clarifying this one, let's make it right.